planet/src/main/webapp/places.js

```diff
@@ -315,9 +342,32 @@ function savePlace(x) {
     if(x.innerHTML === "favorite_border") {
         // Set as saved
         x.innerHTML = "favorite";
+        // Save place to database
+        let name = document.getElementById('place-name').innerHTML; 
+        let address = document.getElementById('place-address').innerHTML; 
+        let open = document.getElementById('openingTime').innerHTML;
+        let close = document.getElementById('closingTime').innerHTML;
+        updateDatabase('test', name, address, open, close); 
```

Comment:
Looks like there's still some debugging code here with the hardcoded 'test' string.

Reply:
Yes! I'm still working on how to access the PlaceID from just a card with details displayed on it. This will be updated in a follow up PR